addons/survey/static/src/js/survey.js

```diff
@@ -21,6 +28,7 @@ if(!the_form.length) {
     var scores_controller = the_form.attr("data-scores");
     var print_mode = false;
     var quiz_correction_mode = false;
+    var ready_with_locale = $.when(base.ready(), load_locale());
```

Comment:
we need to load moment locale as per website language so datetimepicker can display date/calendar in current website language/format
need to wait until locale load ref: https://github.com/odoo/odoo/pull/11590/files/cdf7b75c6b456e7bb817a9b36b3aeaab7e4a5c97#diff-55a36bd725045e96a61a869da9487453R63